### DISCUSSION TITLE
Video speed

### DISCUSSION THREAD

**Discussion by Suqet** [bttv-android/bttv#435] [Ideas]
Could you please add video speed for vod?

**Comment by FoseFx** [comment]
Hey, Thank you for your interest in the project!
I think this is not in the [scope if the project](https://github.com/bttv-android/bttv/blob/master/CONTRIBUTING.md#scope-of-the-project) though.

**Comment by aabithly** [comment]
> Could you please add video speed for vod?
> 

Xtra for twitch has vod speed but its only on github
